docs/authoring.md

```diff
@@ -72,6 +72,125 @@ Group Project XBlock v2 uses standard Studio editing capabilities:
+
+Next select `View ->`, to begin adding components to our Group Project.  We will first add a Group Project Navigator,
+then an activity, and finally, a section for class discussions
+
+
+### Creating the Group Project Navigator
+
+Here we see more details about the validation problems mentioned earlier:
+"Group Project must contain Project Navigator Block."
+
+<img src="/docs/images/walkthrough3.png" width="600" alt="Screenshot of the edit page for an empty gp-v2-project">
+
+So let's resolve that now, by adding a new "Group Project Navigator" component.  elect the button of that name from the
+"Add New Component" section, then click `View ->` to add content to the navigation section.  Here we are given several
+options.  We must add a "Navigation View" to the "Group Project Navigator."  Note that you may need to refresh the page
+to resolve the validation error.  This will be the case on several of the pages that follow.  Now we will also add the
+other four available components:  The "Resources View" will allow learners to see downloadable content we provide for
+them.  "Submissions View" will let them see activities that they have submitted.  Finally, the "Ask a TA View" and
+"Private Discussion View" components will give learners a place to interact with their course staff and with each other
+respectively.  The "Private Discussion View" now has a validation issue, but this will be resolved later.
+
+<img src="/docs/images/walkthrough4.png" width="600" alt="Screenshot of the Group Project Navigator Page, after adding all components">
+
+That concludes our setup of the navigation section.  Now we will add some content to the group project.
+
+
+### Adding an activity.
+
+The Group Project v2 system allows authors to organize student work into distinct activities, each of which can have
+one or more stages required to complete the activity.
+
+To get started, select the "Group Project Activity" button under "Add New Component.  The screen will now look like
+this:
+
+<img src="/docs/images/walkthrough5.png" width="600" alt="Screenshot of the Group Project edit page, with a new Activity">
+
+Now select `View->` in the new "Group Project Activity" panel to begin adding content to the activity.  The components
+we can add to an activity are called "stages."  We will add four stages to our activity: "Text," "Deliverable," "Peer
+Grading," and "Grade Display."  Each one will need further content.
+
+<img src"/docs/images/walkthrough6.png" width="600" alt="Screenshot of the Activity page, with stages added">
+
+In a "Text" stage, we can add HTML, which will be displayed to our learners, as well as a few other kinds of content
+blocks.  For this demo, we'll just add a single HTML snippet, as shown here.
+
+<img src"/docs/images/walkthrough7.png" width="600" alt="Added some text.">
+
+
+Then, to add a stage where users upload files to complete their work, we can add a "Submission," clicking first the
```

Comment:
This is a bit confusing. I think you actually need to click on the "Deliverable" button to add a Submission stage.